packages/web/src/stores/user.ts

```diff
@@ -31,5 +32,8 @@ export const useUserStore = create<UserStore>((set) => ({
       }
     }
     set({ user, doneLoading: true });
+
+    // Load user-dependent stores
+    void useAdminStore.getState().fetchAdmin();
```

Comment:
what does this void do in this context?

Reply:
Basically says "don't wait on this admin fetch to complete; just trigger it and move on"

Reply:
It's the same as leaving nothing but `eslint` wants us to pick a lane so we don't forget to await a promise